### ISSUE TITLE
Enhancement: Tab overflow to rows instead of scroller

### ISSUE THREAD

**Issue by dasjestyr** [ButchersBoy/Dragablz#69]
The base version of the tabs control will create tab rows by default as the tabs overflow the window. I believe this is an important thing to have when working with an application with many tabs (especially if the user is the one creating the tabs) and the tabs need to be visible and accessed rapidly. Having this as an optional behavior would be very useful.

**Comment by shawnmanderson** [comment]
This feature would be a great addition, and I'd love to see this one. I've already implemented my own custom tab drop down solution (i.e. #45), but having tab rows would be a better solution.

**Comment by ButchersBoy** [comment]
Yes...I would like to do this..at some point...but it has it's complexities so there are other issues I will be tidying up first.

**Comment by Lilium-Candidum** [comment]
May I ask what the status for this feature is?
